package.json

```diff
@@ -120,7 +120,7 @@
 		"fast-glob": "2.2.7",
 		"fbjs": "0.8.17",
 		"glob": "7.1.2",
-		"husky": "3.0.5",
+		"husky": "2.7.0",
```

Comment:
Was it intentional to downgrade? I would expect it would need corresponding changes to `package-lock.json` if intended.

Reply:
I need to revert this. 

For context: when [we updated husky from 0 to 3](https://github.com/WordPress/gutenberg/pull/17310/files) we [introduced](https://github.com/typicode/husky/releases/tag/v3.0.0) a breaking change:  the git version required to work with husky is `>= 2.13.2`. Git hooks won't be executed in versions below. The git version bundled with my OS is lower than that, so I can't run the hooks (and test this change), so I had to downgrade it for testing.

I asked in core-editor about this change and it doesn't seem a widespread issue. Wasn't able to pin down easily the git versions that come with the supported OS for Windows and Mac to gauge how many could be affected. I guessed another way to look at it was that if people don't complain about hooks not executing for them, it's not an issue.

Reply:
31378db